pom.xml

```diff
@@ -93,9 +93,11 @@
         <caf.container.processing.service.name>${dockerDataProcessingOrg}processing-service${dockerProjectVersion}</caf.container.processing.service.name>
         <caf.corepolicy.version>1.0.0-4</caf.corepolicy.version>
         <caf.worker.policy.version>1.4.0-60</caf.worker.policy.version>
+        <cafWorkerDocumentFrameworkVersion>3.3.0-SNAPSHOT</cafWorkerDocumentFrameworkVersion>
```

Comment:
OK. but just on that property, do you not think having the project versions in a single section is useful for when updating the pom?

Reply:
Having them in a single section makes sense.  I think the best structure is that one that the [Tag Service](https://github.houston.softwaregrp.net/Verity/tag-service) uses.  It has a dependency management section in the top-level aggregator project, and all the dependencies used by all of its modules are listed in it.  That even includes the modules themselves in case there are dependencies between them.  When the dependencies are actually referenced in the modules there is no need for them to reference a version number at all.  Really the only point I can see of having the version number in a property is if either a) you also need the version number for some other purpose, like to include in a Docker label or something like that, or b) you want to be able to change the version from the command line (unusual).